pkg/signup/signup_callback_handler.go

```diff
@@ -1,34 +0,0 @@
-package signup
-
-import (
-	"encoding/json"
-	"github.com/codeready-toolchain/registration-service/pkg/configuration"
-	"github.com/gin-gonic/gin"
-	"log"
-	"net/http"
-)
-
-// SignupCallbackService implements the signup callback endpoint.
-type SignupCallbackService struct {
```

Comment:
Why was this file deleted?  This handler is intended to be the callback endpoint for which the SSO authentication will redirect the user to.

Reply:
@sbryzak i had commented on your PR about the flow from discussing with @alexeykazakov . It can be found here: https://github.com/codeready-toolchain/registration-service/pull/13#discussion_r322421842 we don't need the SignupCallbackService..